RESTserver/src/main/java/com/softserve/if072/restservice/service/security/AnalyticsSecurityService.java

```diff
@@ -0,0 +1,23 @@
+package com.softserve.if072.restservice.service.security;
+
+import com.softserve.if072.restservice.service.ProductService;
+import org.springframework.stereotype.Service;
+
+/**
+ * The ForecatsSecurityService class provides security methods for AnalyticsControlLer
+ *
+ * @author Igor Kryviuk
+ */
+@Service
+public class AnalyticsSecurityService extends BaseSecurityService {
+    private final ProductService productService;
+
+    public AnalyticsSecurityService(ProductService productService) {
+        this.productService = productService;
+    }
+
+    public boolean hasPermissionToAccess(int productId) {
+        int productOwnerId = productService.getUserIdByProductId(productId);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "productOwnerId". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused "productOwnerId" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1481)